src/interfaces/store-interfaces/state-interfaces/flags-review-certify-interfaces/alteration-flags-review-certify.ts

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Each of the flags are now separated preventing incorrect scroll behavior for incomplete filings